File: Assignment 13/search_element.c
//Write user defined function to search an element an element in an integer array with pointer as argument
#include<stdio.h>
int searchElement(int len, int element, int *ptr);
int main(){
	int arr[]={1,0,6,3,2,4,9,8,7,5};
        int len= sizeof(arr)/sizeof(arr[0]);
	int element;
	printf("Enter element to be searched : ");
	scanf("%d", &element);
	searchElement(len, element, arr);
	return 0;
}
int searchElement(int len, int element, int *ptr){
	int i, cnt=0;
	for(i=0;i<len;i++){
		if(*(ptr+i)==element){
			cnt++;
		}
		else{
			continue;
		}
	}
	if(cnt==0){
		printf("Element not found \n");
	}
	else{
		printf("Element found \n");
	}
}

File: 2nd Year/3rd semester/Assignment 2/SearchElement.java
//Write a program that declares an array of n integers and searches an inputted element is present in the array or not. If present, then display the position of the element in the array

import java.util.*;

class SearchElement{
	public static void main(String[] args){
		int n, element, cnt=0;
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter number of elements : ");
		n = sc.nextInt();
		int arr[] =new int[n];
		System.out.print("Enter " +n+ " values : ");
		for (int i= 0; i<n; i++){
			arr[i] = sc.nextInt();
		}
		System.out.print("Enter element to be searched : ");
		element = sc.nextInt();
		for(int i=0;i<arr.length;i++){
			if(element == arr[i]){
				cnt++;
				System.out.println(element +" is found at position " + (i+1));
				break;
			}
			else{
				continue;
			}
		}
		if(cnt==0){
			System.out.println("Element not found...");
		}

	}
}
